### PR TITLE
GetPreparationKitFiles Function

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

File: config.ts [hunk 1]
```diff
@@ -89,4 +89,7 @@ export default {
             .asString()
             .split(','),
     },
+    preparationKit: {
+        preparationKitBlobName: 'preparation-kit',
```
Comment (carmeltsadok) [inline config.ts:93]: Get from env

File: GetPreparationKitFiles/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { httpErrorHandler, ForbiddenFunctionError, ObjectNotFoundFunctionError } from '../shared/error/error';
+import { Role, User } from '../shared/user/user.interface';
+import HttpTriggerResult from '../shared/httpTriggerResult';
+import config from '../config';
+import getBlockBlobClient from '../shared/blob/blob.client';
+import authWrapper from '../shared/authWrapper';
+
+const functionName = 'GetPreparationKitFiles';
+
+const GetPreparationKitFiles: AzureFunction = async (context: Context, req: HttpRequest, user: User): Promise<HttpTriggerResult> => {
```
Comment (carmeltsadok) [inline GetPreparationKitFiles/index.ts:11]:
```suggestion
const GetPreparationKitFiles: AzureFunction = async (context: Context, _req: HttpRequest, user: User): Promise<HttpTriggerResult> => {
```

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: config.ts (hunk 1 above)
Comment (PCOffline) [reply config.ts:93]: According to Adar, we set a convention that such strings can be hard-coded

Comment (carmeltsadok) [reply config.ts:93]: this config file gets the variables from the env.. I think you should add it with a default value (like other blob names in this file)

**Review by carmeltsadok — APPROVED** [review]

**Review by lustigy96 — APPROVED** [review]

**Pull request merged**